scripts/qemu-check.sh

```diff
@@ -1,8 +1,8 @@
-#!/bin/bash
 # SPDX-License-Identifier: BSD-3-Clause
 # Copyright(c) 2018 Intel Corporation. All rights reserved.
+#!/bin/bash
 
```

Comment:
Is this an accidental change? We don't need to move shebang string around.

Reply:
I think it's in fact a breaking change if we move it around. So yeah, the shebang should remain on the first line.